src/mds/Server.cc

```diff
@@ -5131,6 +5265,32 @@ void Server::create_quota_realm(CInode *in)
   mds->send_message_mds(req, in->authority().first);
 }
 
+void Server::create_user_quota_realm(CInode *in)
+{
+  dout(10) << __func__ << " " << *in << dendl;
+
+  auto req = MClientRequest::create(CEPH_MDS_OP_SETXATTR);
+  req->set_filepath(filepath(in->ino()));
+  req->set_string2("ceph.user_quota");
+  // empty vxattr value
+  req->set_tid(mds->issue_tid());
+
+  mds->send_message_mds(req, in->authority().first);
+}
+
+void Server::create_group_quota_realm(CInode *in)
+{
+  dout(10) << __func__ << " " << *in << dendl;
+
+  auto req = MClientRequest::create(CEPH_MDS_OP_SETXATTR);
+  req->set_filepath(filepath(in->ino()));
+  req->set_string2("ceph.group_quota");
+  // empty vxattr value
+  req->set_tid(mds->issue_tid());
+
+  mds->send_message_mds(req, in->authority().first);
+}
```

Comment:
No reason for introducing these  functions. using Server::create_quota_realm() should be enough

Reply:
fixed it ,thanks.